mdanalysissandbox/bruce.py

```diff
@@ -0,0 +1,19 @@
+""" this is my code module
+"""
+
+import numpy as np
+
+def cound_names(ag, name):
+	"""count how many atoms have a particular name
+
+	parameters
+	-----------
+	ag : Atom group
+	name : str
+		the name to look for
+
+	Returns
+	-------
+	count :int
+	"""
+	return len(ag[ag.names ==name])
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/) says to have spaces around operators.
```python
ag.names == name
```